refinery/data_set_manager/utils.py

```diff
@@ -806,10 +800,17 @@ def format_solr_response(solr_response):
         solr_response_json = json.loads(solr_response)
     except TypeError:
         return "Error loading json."
+    assert solr_response_json['responseHeader']['status'] == 0, \
```

Comment:
I'm not on the same page right now, but this line shouldn't block the PR. I'll revert it; meanwhile, I've filed #1789 to try to explain where I'm coming from. Sorry for the distraction.